src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
         });
 
+        attrs.$observe( 'tourtipVerticalOffset', function ( val ) {
```

Comment:
Would you mind renaming the `verticalOffset` and `horizontalOffset` properties to `offsetVertical` and `offsetHorizontal` just so the properties' namings align a little better?